extensions/3DTILES_metadata/0.0.0/schema/3DTILES_metadata_implicit_tiling_object.schema.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Depending on the answer to the other question, we might want to merge this file with `instanceTable.property.schema.json`, just not sure which is the best way to do that yet.